lib/tools.js

```diff
@@ -52,9 +53,16 @@ function genLog (level, hook) {
         msg = n.shift()
         formatParams = n
       }
+      if (typeof this[msgPrefixSym] === 'string' && msg !== undefined && msg !== null) {
+        msg = this[msgPrefixSym] + msg
```

Comment:
I think this is one of the cases where a coercive check is preferred:

```suggestion
      if (typeof this[msgPrefixSym] === 'string' && msg) {
        msg = this[msgPrefixSym] + msg
```

Reply:
thanks, done